maven-plugin/pom.xml

```diff
@@ -6,30 +6,35 @@
     <version>1.27-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
-  <artifactId>maven-localizer-plugin</artifactId>
+  <artifactId>localizer-maven-plugin</artifactId>
   <packaging>maven-plugin</packaging>
   <name>Localizer generator</name>
 
+  <prerequisites>
+    <!--  3.3.1 is the lowest maven requiring java7 -->
+    <maven>3.3.1</maven>
```

Comment:
the generator and plugin are now compiled with Java7 (anyone using less that that can continue to use the older version -but java7 should be a safe minimum bar now, along with maven 3.3.1 which is also relatively old)